design-toolkit/components/src/index.css

```diff
@@ -240,6 +240,13 @@
   --icon-color: --value(--color- *);
 }
 
+/* Table Related Custom Variants */
```

Comment:
These should be mixed in with the other custom variants. The order in which they are in this file is important and dictates. Also, I had an open question about what `active` represents and if we already have a selector that aligns with its purpose